src/test/scala/mesosphere/marathon/core/task/TaskTest.scala

```diff
@@ -156,26 +156,6 @@ class TaskTest extends UnitTest with Inside {
       task.isUnreachableExpired(f.clock.now, 10.minutes) should be(false)
     }
 
-    "a reserved task transitions to launched on running MesosUpdate" in {
```

Comment:
reserved isn't gone yet

Reply:
I don't think an instance can have `Condition.Reserved`.

Reply:
it can, because removing that would mean we would have to add migration to Phase 2 as well, which I did not want to do

Reply:
Good point.